modules/RTC/JitsiTrack.js

```diff
@@ -60,10 +60,9 @@ function addMediaStreamInactiveHandler(mediaStream, handler) {
  *        onended/oninactive events of the stream.
  * @param trackMediaType the media type of the JitsiTrack
  * @param videoType the VideoType for this track if any
- * @param ssrc the SSRC of this track if known
  */
-function JitsiTrack(conference, stream, track, streamInactiveHandler, trackMediaType,
-                    videoType, ssrc)
+function JitsiTrack(conference, stream, track, streamInactiveHandler,
```

Comment:
Ssrc is no longer part of the JitsiLocalTrack, but only of JitsiRemoteTrack. For local tracks SSRCs have to be obtained through the TraceablePeerConnection. I guess it could still be stored in JitsiLocalTrack, but would have to be mapped on per peer connection basis.